Library/Homebrew/extend/os/linux/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,37 @@
+require "requirement"
+
+class OsxfuseRequirement < Requirement
+  download "https://github.com/libfuse/libfuse"
```

Comment:
Very much so, but my understanding was you preferred we not include that. Did you change your mind or am I just misunderstanding your previous comments?